examples/chip-tool/README.md

```diff
@@ -91,3 +91,11 @@ To get the list of parameters for a specific command, run the built executable
 with the target cluster name and the target command name
 
     $ chip-tool onoff on
+
+### How to parse a setup code
+
+To parse a setup code, run the built executable with the `parse` cluster name
+and the target command name
+
+    $ chip-tool parse qr base41-representation
```

Comment:
can you provide a few example strings here for that others can use to test?

Reply:
Example for QRCode, Manual code, and QRCode with TLV data added in the new revision.